src/Microsoft.ML.ImageAnalytics/MLImage.cs

```diff
@@ -126,6 +126,16 @@ private set
             }
         }
 
+        public ReadOnlySpan<byte> PixelsNoAlpha
+        {
+            get
+            {
+                ThrowInvalidOperationExceptionIfDisposed();
+                var i = _image.Copy(SKColorType.Rgb565);
```

Comment:
Why do you need this line?

Reply:
That was in there for testing. Its been removed now.